python/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# common files to all platforms
+ly_install_files(
+    FILES
+        get_python.cmake
+        readme.md
+        requirements.txt
+    DESTINATION python
+)
+
+# platform specific files (they are PROGRAMS)
+include(Platform/${PAL_PLATFORM_NAME}/install_files_${PAL_PLATFORM_NAME_LOWERCASE}.cmake)
```

Comment:
Since the INSTALL command is available for Android and iOS we should need to have an `Platform/Android/install_files_android.cmake` and `Platform/iOS/install_files_ios.cmake` with an empty list of files to install.

Otherwise this will fail to configure on those Platforms

Reply:
will add a `if(PAL_TRAIT_BUILD_HOST_TOOLS)` instead